propertyestimator/backends/backends.py

```diff
@@ -4,7 +4,7 @@
 import re
 from enum import Enum
 
-from simtk import unit
+from propertyestimator import unit as unit
```

Comment:
In most of the other files, this doesn't have the `as unit` part at the end. Does that make a difference?

Reply:
Typo - thanks!